demo/encodermlp-config.yaml

```diff
@@ -0,0 +1,29 @@
+encoder:
+  nr_image_channels: 1
+  channels_list:
+  - 32
+  - 64
+  - 128
+  - 256
+  - 512
+  - 512
+  - 512
+mlp:
+  channels_list:
+  - 512
+  - 512
+  - 512
+
+
+loss:
+  ntxent_temperature: 0.1
```

Comment:
Is it weird that we have a general model repo that introduce an encoder-mlp but already assumes a contrastive loss? The tutorial part for this could just use normal cross entropy or even MSE?

Reply:
I don't think it's weird, since the contrastive loss is part of the models repo/package? The tutorial showcases how to use this loss function.